tools/accuracy_checker/accuracy_checker/postprocessor/resize.py

```diff
@@ -0,0 +1,172 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from functools import singledispatch
+import cv2
+import numpy as np
+from ..representation import (
+    DepthEstimationAnnotation, DepthEstimationPrediction,
+    SegmentationPrediction, SegmentationAnnotation,
+    StyleTransferAnnotation, StyleTransferPrediction,
+    SuperResolutionPrediction, SuperResolutionAnnotation)
+from ..postprocessor.postprocessor import PostprocessorWithSpecificTargets
+from ..config import NumberField
+from ..utils import get_size_from_config
+
+try:
+    from PIL import Image
+except ImportError:
+    Image = None
+
+
+class Resize(PostprocessorWithSpecificTargets):
+
+    __provider__ = 'resize'
+
+    prediction_types = (DepthEstimationPrediction, StyleTransferPrediction,
+                        SegmentationPrediction, SuperResolutionPrediction, )
+    annotation_types = (DepthEstimationAnnotation, StyleTransferAnnotation,
+                        SegmentationAnnotation, SuperResolutionAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination width for resize"
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination height for resize."
+            ),
+            'size': NumberField(
+                value_type=int, optional=True, min_value=1,
+                description="Destination size for resize for both dimensions (height and width)."
+            )
+        })
+        return parameters
+
+    def configure(self):
+        if Image is None:
+            raise ValueError('{} requires pillow, please install it'.format(self.__provider__))
+        self.dst_height, self.dst_width = get_size_from_config(self.config, allow_none=True)
+
+    def process_image(self, annotations, predictions):
+        if self.dst_width and self.dst_height:
+            height = self.dst_height
+            width = self.dst_width
+        else:
+            height = self.image_size[0]
+            width = self.image_size[1]
+
+        @singledispatch
+        def resize(entry, height, width):
+            return entry
+
+        @resize.register(DepthEstimationPrediction)
+        def _(entry, height, width):
+            entry.depth_map = cv2.resize(entry.depth_map, (width, height))
+
+            return entry
+
+        @resize.register(StyleTransferAnnotation)
```

Comment:
style transfer resize and super resolution resize do the same, you can just specify several decorators instead of different methods
```suggestion
        @resize.register(StyleTransferAnnotation)
        @resize.register(SuperResolutionAnnotation)
```

Reply:
Done